.github/workflows/pr-dependabot.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Check out code
-      uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # tag=v3.5.3
+      uses: actions/checkout@v4 # tag=v3.5.3
```

Comment:
I'll do a follow-up to specify the tag & sha on all actions

If this is done correctly actions are pinned by sha while still preserving (& auto-bumping) the tag info